src/Extension/Tasks/SprykAddModuleTask.yaml

```diff
@@ -3,28 +3,28 @@ id: "generate:php:add-module"
 short_description: "Generate a new Spryker module in the current project"
 help: ~
 stage: build
-command: "APPLICATION_ENV=development CORE_NAMESPACES=%coreNamespaces% PROJECT_NAMESPACES=%projectNamespaces% PROJECT_NAMESPACE=%projectNamespace% php %sdk_dir%/vendor/bin/spryk-run AddModule --mode=%coreLevel% --organization=%namespace% --module=%module%"
+command: "php %sdk_dir%/vendor/bin/spryk-run AddModule --mode=%coreLevel% --organization=%namespace% --module=%module%"
```

Comment:
Why do we have such a low-level Spryk in use here? It doesn't make sense to expose those ones.

Reply:
it's for example. we have others command like this